src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supplyResult = 0;
+        int buyResult = 0;
+        File fromFile = new File(fromFileName);
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile));
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] splitLine = line.split(",");
+                if (splitLine[0].equals("supply")) {
+                    supplyResult += Integer.parseInt(splitLine[1]);
```

Comment:
Here you have code duplication. We can parse value before entering to `if-else` statement